### ISSUE TITLE
Plugin not enabling at server start-up

### ISSUE THREAD

**Issue by Zatiejo** [mcMMO-Dev/mcMMO#4549]
Hi.

I'm not very smart with server error logs or messages. So, when I saw this, I wasn't sure if I should report it or not. There's probably a closed/open ticket or issue on the same topic, but as I keep reading through, my eyes and brain aren't really able to tell if it's a similar error or issue. So, if you find this as a duplicate of something that's already been solved or opened, then, my apologies.

Here's a pastebin link to the error logs I got in my console. Let me know in advance if this pastebin link is not accessible or working.
https://pastebin.com/qLYUFtMa

The start-up (or enabling plugin) messages only appear when the server starts up or starts enabling plugins in order for them to load. This does not spam or fill up the console. This error occurs only once.

This message, when a player joins, spams constantly at any action the player does (or from what I see, everytime I move, do an action such as digging, etc...) it always spams this same error message, (perhaps slightly different, depending on the action,) constantly.

Here are my current plugins. Some of these I think could be factors to this error- but if not, then just simply ignore these thoughts:

AntiCooldown 3.6.4.1
DeathMessages 1.3.6
EconomyShopGUI 3.3-b.1
EssentialsX (Main, Chat and Spawn Plugins) 2.19.0-dev+162-780aade (could be a factor or cause due to it being a dev release)
GriefPrevention 16.17.1 
GSit 5.3.2.0
LPC 3.2.0
LuckPerms 5.3.47
MarriageMaster 2.4
**mcMMO 2.1.198**
minimotd 2.0.2
nuvotifier 2.7.2
PlaceHolderAPI 2.10.9 (could be a cause since it's up to date, but doesn't support 1.17)
PlayerReport 3.4.1
Rankup 3.11.3 
Vault 1.7.3 (could be a cause, similar to PlaceHolderAPI reason)
VotingPlugin 6.4
WorldEdit 7.2.6-SNAPSHOT-dist
WorldGuard 7.0.6-SNAPSHOT-dist

Spigot Version - 1.17.0

This is as much info as I can give (for the moment.) This isn't too urgent, so take your time :)

**Comment by Zatiejo** [comment]
Correction: PlaceHolderAPI (and possibly another plugin) were causing these issues, sorry! I just reloaded the server with fresh files and it's not the same as before- mcmmo is now working just fine. Sorry.
